### ISSUE TITLE
sass 3.3.0 does not appear to --user-install

### ISSUE THREAD

**Issue by dracos** [sass/sass#1153]
Firstly, sass is lovely and I'm very glad it exists, thanks for all the work that has been done on it :)

On the current version of Debian, with the ruby1.9.1 and ruby1.9.1-dev packages installed (note that that means it's ruby 1.9.3 and gem 1.8.23) and no existing `~/.gem/` directory, I get the following error when trying to install sass 3.3:

```
$ gem install --user-install sass -v 3.3.0
Fetching: rake-10.1.1.gem (100%)
WARNING:  You don't have /home/matthew/.gem/ruby/1.9.1/bin in your PATH,
      gem executables will not run.
Fetching: sass-3.3.0.gem (100%)
Building native extensions.  This could take a while...
ERROR:  Error installing sass:
    ERROR: Failed to build gem native extension.

        /usr/bin/ruby1.9.1 mkrf_conf.rb
/usr/lib/ruby/1.9.1/fileutils.rb:247:in `mkdir': Permission denied - /var/lib/gems (Errno::EACCES)
    from /usr/lib/ruby/1.9.1/fileutils.rb:247:in `fu_mkdir'
    from /usr/lib/ruby/1.9.1/fileutils.rb:221:in `block (2 levels) in mkdir_p'
    from /usr/lib/ruby/1.9.1/fileutils.rb:219:in `reverse_each'
    from /usr/lib/ruby/1.9.1/fileutils.rb:219:in `block in mkdir_p'
    from /usr/lib/ruby/1.9.1/fileutils.rb:205:in `each'
    from /usr/lib/ruby/1.9.1/fileutils.rb:205:in `mkdir_p'
    from /usr/lib/ruby/1.9.1/rubygems/installer.rb:447:in `verify_gem_home'
    from /usr/lib/ruby/1.9.1/rubygems/installer.rb:147:in `install'
    from /usr/lib/ruby/1.9.1/rubygems/dependency_installer.rb:297:in `block in install'
    from /usr/lib/ruby/1.9.1/rubygems/dependency_installer.rb:270:in `each'
    from /usr/lib/ruby/1.9.1/rubygems/dependency_installer.rb:270:in `each_with_index'
    from /usr/lib/ruby/1.9.1/rubygems/dependency_installer.rb:270:in `install'
    from mkrf_conf.rb:18:in `<main>'
Ensuring you have the right version of listen installed.
Installing listen >= 1.1.0, < 3.0.0
```

sass 3.2.14 is fine:

```
$ gem install --user-install sass -v 3.2.14
Fetching: sass-3.2.14.gem (100%)
WARNING:  You don't have /home/matthew/.gem/ruby/1.9.1/bin in your PATH,
      gem executables will not run.
Successfully installed sass-3.2.14
1 gem installed
Installing ri documentation for sass-3.2.14...
Installing RDoc documentation for sass-3.2.14...
```

Why is a `--user-install` trying to make a directory in `/var/lib/gems`? Even if I install the listen gem manually with `gem install --user-install listen`, which does successfully install, then it still gives the same issue, saying it's installing listen and not being able to write globally.

Hope that's helpful.

**Comment by dracos** [comment]
Done some more investigation, looks like this was/is a bug in rubygem that was fixed in version 2.1.2 (from rubygems/rubygems#642, though that says it was only introduced in 2.1.0, but hey) so guess I have to work out how to use a non-packaged version of gem. Sorry for the noise. Though it would be good if it could easily be made to work somehow :)

**Comment by dracos** [comment]
For anyone else running on Debian who's interested, in the end I did this:

```
$ gem install --user-install bundler
[...]
$ echo -e "source 'https://rubygems.org'\ngem 'sass'" > Gemfile
$ .gem/ruby/1.9.1/bin/bundle install --path /home/matthew/.gem
[...]
$ .gem/ruby/1.9.1/bin/sass --version
Sass 3.3.0 (Maptastic Maple)
```
